compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -209,6 +210,9 @@ void CircleOptimizer::quantize(loco::Graph *g) const
     luci::QuantizeWithMinMaxPass quantizer(str_to_dtype(input_dtype), str_to_dtype(output_dtype),
                                            str_to_granularity(granularity));
     quantizer.run(g);
+
+    luci::PropagateConcatenationQparamPass propagater;
```

Comment:
(minor) couldn'tfind `propagater` but `propagator`

Reply:
Typo.. I'll change the name in the following PRs. :)